pages/_app.js

```diff
@@ -5,6 +5,7 @@ import '../styles/globals.css';
 const Application = ({ Component, pageProps }) => {
   return (
```

Comment:
The Header component is used but not imported. This will cause a ReferenceError.

Suggestion:
```javascript
import Header from '../components/Header';
```